openmmforcefields/generators/system_generators.py

```diff
@@ -206,7 +206,7 @@ def __init__(self, forcefields=None, small_molecule_forcefield='openff-1.0.0', f
                     _logger.debug(f'Trying {template_generator_cls.__name__} to load {small_molecule_forcefield}')
                     self.template_generator = template_generator_cls(forcefield=small_molecule_forcefield, cache=cache, template_generator_kwargs=self.template_generator_kwargs)
                     break
-                except ValueError as e:
+                except (ValueError, NotImplementedError) as e:
```

Comment:
This is to catch the way I nuked `GAFFTemplateGenerator`, probably something that should be reverted alongside adding it back